content/news/posts/2024/prestashop-news-where-we-are-and-where-we-are-going.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  Prestashop News - Where We Are And Where We’re Going
+subtitle: "I don't know what to write here"
+date:   2024-01-04
```

Comment:
I put today's date but if we prefer to publish tomorrow, feel free to change this

Reply:
Good for me!